pylsp/config/schema.json

```diff
@@ -370,12 +370,12 @@
       "description": "Enable or disable the plugin."
     },
     "pylsp.rope.extensionModules": {
-      "type": ["null", "string"],
+        "type": ["string", "null"],
       "default": null,
       "description": "Builtin and c-extension modules that are allowed to be imported and inspected by rope."
     },
     "pylsp.rope.ropeFolder": {
-      "type": ["null", "array"],
+      "type": ["array", "null"],
```

Comment:
Note, I re-ordered the array values here to be consistent with the other type fields.